src/ImageViewerCaptureTool.cpp

```diff
@@ -19,16 +19,19 @@ ImageViewerCaptureTool::ImageViewerCaptureTool(uint width, uint height) {
 }
 
 ImageViewerCaptureTool::ImageViewerCaptureTool(double fovY, double fovX, uint value, bool isHeight) {
-    double aspectRatio = fovX / fovY;
     uint width, height;
+    double fovYrad = fovY * M_PI / 180.0;
```

Comment:
Ok..
it was my mistake..


On Thu, May 12, 2016 at 6:09 PM, Sylvain Joyeux <notifications@github.com>
wrote:

> In src/ImageViewerCaptureTool.cpp
> <https://github.com/Brazilian-Institute-of-Robotics/gui-vizkit3d_normal_depth_map/pull/7#discussion_r63048562>
> :
>
> >      uint width, height;
> > +    double fovYrad = fovY * M_PI / 180.0;
>
> Change. No degrees in Rock/FlatFish code. If some external APIs need
> degrees, you do the conversion when calling that API.
>
> In general, never mix different representations of the same quantities.
> Angles in Rock are in radians, don't start having mixes of degrees and
> radians. No exceptions.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Brazilian-Institute-of-Robotics/gui-vizkit3d_normal_depth_map/pull/7/files/5bd3e5ec9ab01d8457a28780b04e588989bde62f#r63048562>
>



-- 
§ 2010 Tiago Trocoli